qt-bugs@ issue : 153183
bugs.kde.org number : 
applied: yes
author: woebbeking@kde.org

In Qt 3.3.8 QDateTime::fromString() has an off by one regression:
- Thu Mar  3 19:45:58 2005 -> invalid QDateTime
- Thu Mar 13 19:45:58 2005 -> Mar 1 2005
 
--- src/tools/qdatetime.cpp
+++ src/tools/qdatetime.cpp
@@ -2485,7 +2485,7 @@
 	    return dt;
 	}
 #endif
-	int day = s.mid( firstSpace + 4, 2 ).simplifyWhiteSpace().toInt();
+	int day = s.mid( firstSpace + 5, 2 ).simplifyWhiteSpace().toInt();
 	int year = s.right( 4 ).toInt();
 	QDate date( year, month, day );
 	QTime time;
